### DISCUSSION TITLE
Implement zstd as an additional archive format over tgz for vercel deploy

### DISCUSSION THREAD

**Discussion by shouze** [vercel/vercel#9088] [Ideas]
zstd compress better & faster :zap: 

Github actions have already managed that [so this is a good source of inspiration](https://github.com/actions/toolkit/blob/819157bf872a49cfcc085190da73894e7091c83c/packages/cache/src/internal/tar.ts).

https://github.com/vercel/vercel/blob/dbc00a7c3e4ddfb63d712fd960448455e6bd206d/packages/client/src/types.ts#L15

https://github.com/vercel/vercel/blob/dbc00a7c3e4ddfb63d712fd960448455e6bd206d/packages/cli/src/commands/deploy/index.ts#L549C32-L559

https://github.com/vercel/vercel/blob/dbc00a7c3e4ddfb63d712fd960448455e6bd206d/packages/cli/src/util/deploy/create-deploy.ts#L11-L30

https://github.com/vercel/vercel/blob/dbc00a7c3e4ddfb63d712fd960448455e6bd206d/packages/cli/src/util/index.ts#L159C39-L177

https://github.com/vercel/vercel/blob/dbc00a7c3e4ddfb63d712fd960448455e6bd206d/packages/cli/src/util/deploy/process-deployment.ts#L79-L104

https://github.com/vercel/vercel/blob/dbc00a7c3e4ddfb63d712fd960448455e6bd206d/packages/client/src/create-deployment.ts#L93-L114

**Comment by amyegan** [comment]
Thanks for sharing this idea! I added it to the list of feature requests 😄
